qualtricssurvey/private/view.js

```diff
@@ -0,0 +1,11 @@
+function QualtricsSurveyView(runtime, element) {
```

Comment:
Does this mean everything related to JavaScript (including JSHint) should be removed from the Gruntfile?